src/xml/zcl_abapgit_xml.clas.testclasses.abap

```diff
@@ -0,0 +1,90 @@
+CLASS ltcl_xml DEFINITION DEFERRED.
+
+CLASS ltcl_xml_concrete DEFINITION FOR TESTING
+    FINAL
+    INHERITING FROM zcl_abapgit_xml
+    FRIENDS ltcl_xml.
+ENDCLASS.
+
+CLASS ltcl_xml_concrete IMPLEMENTATION.
+ENDCLASS.
+
+CLASS ltcl_xml DEFINITION FOR TESTING DURATION SHORT RISK LEVEL HARMLESS.
+
+  PRIVATE SECTION.
+
+    METHODS:
+      space_leading_trailing FOR TESTING
+        RAISING zcx_abapgit_exception.
+
+    METHODS:
+      parse_xml
+        IMPORTING
+          iv_xml TYPE csequence
+        RAISING
+          zcx_abapgit_exception,
+      render_xml
+        IMPORTING
+          iv_name       TYPE string
+        RETURNING
+          VALUE(rv_xml) TYPE string.
+
+    DATA: lo_xml TYPE REF TO ltcl_xml_concrete.
```

Comment:
rename to `mo_xml`, LO_ is for local variables

Reply:
done